recipe/meta.yaml

```diff
@@ -53,22 +55,22 @@ outputs:
         - {{ pin_subpackage("libabseil", max_pin="x") }}
         # also pin on ABI variant
         - libabseil =*=cxx{{ cxx_standard }}*
-        # abseil needs the 10.13 headers even just to be `# include`d;
-        # ensure that all dependent packages reflect that to
-        # avoid the solver getting stuck in no-man's-land.
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
+      ignore_run_exports:
+        - gtest
 
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
         - cmake
         - ninja
+      host:
+        - gtest
```

Comment:
I understand that you want to patch https://github.com/abseil/abseil-cpp/commit/779a3565ac6c5b69dd1ab9183e500a27633117d5, but is there any reason to add gtest as the dependency?

Reply:
That's what I've been trying to figure out as well. At least, with the `ignore_run_exports`, it wouldn't affect the artefacts we produce (in terms of run dependencies).

Reply:
It is not needed, and I have since removed it. The compiler error above was a red herring.